docs/docs/faq.md

```diff
@@ -68,7 +68,7 @@ If you haven't already done so, please read these FAQs first:
 
 When a keyboard accepts a small “PCB MCU module” (e.g. _Arduino Pro Micro_) for its “brains”, then it's important to conceptually separate the hardware into a [board](#what-is-a-board) PCB and a [shield](#what-is-a-shield) PCB.
 
-The [shield](#what-is-a-shield) is a brainless shell containing all the keys, RGB LEDs, encoders etc. It maps all of these features to a standard pin footprint, such as the Pro Micro pinout.
+The [shield](#what-is-a-shield) is a "brainless" shell containing all the keys, RGB LEDs, encoders etc. It maps all of these features to a standard pin footprint, such as the Pro Micro pinout.
```

Comment:
Not sure if we use ironic quotes anywhere else in the documentation. Perhaps we could change out the term entirely?